src/AuthenticatedApp.jsx

```diff
@@ -76,6 +77,16 @@ function AuthenticatedApp() {
 
      
       {user ? (
+
+        <>
+        <Navbar_professional />
+        <Routes>
+            <Route path={"/home"} element={<Search />} />
+            <Route path={"/"} element={<Search />} />
+          </Routes>
+        </>
```

Comment:
You can also use `Outlet` to build a layout that includes the `navbar` , just it a suggestion [Routing Docs - Outlet](https://github.com/remix-run/react-router/blob/f59ee5488bc343cf3c957b7e0cc395ef5eb572d2/docs/installation/getting-started.md#relative-links)